### PR TITLE
Only call focus() if we had it before [issue 495]

### PR DESCRIPTION
**Pull request by joefradley**
To address this [issue](https://github.com/zenoamaro/react-quill/issues/495).

### CODE REVIEW

**Comment by jfradley** [comment]
Hi @alexkrolick , I'm not sure if you're the right person to ping for this but if you could I'd appreciate.

**Comment by alexkrolick** [comment]
👀

**Comment by Felix-Indoing** [comment]
waiting for this to be merged

**Comment by oschwede** [comment]
Also waiting. Additionally the Travis-Check was successfull, but github doesnt see that

**Comment by jfradley** [comment]
What's the best way for me to go about getting this merged?

File: src/component.js [hunk 1]
```diff
@@ -207,8 +207,10 @@ var QuillComponent = createClass({
 		if (this.quillDelta) {
 			this.editor.setContents(this.quillDelta);
 			this.editor.setSelection(this.quillSelection);
```
Comment (oschwede) [inline src/component.js]: According to the quill docs, calling setSelection will also focus the editor. I think `setSelection()` should only be called, if `quillSelection` is not null/undefined.

Comment (jfradley) [reply src/component.js]: > According to the quill docs, calling setSelection will also focus the editor

It doesn't look like it calls focus if the selection (e.g. `quillSelection`) is null/undefined. Regardless, it's probably best not to call it anyway if it's undefined.

**Comment by jfradley** [comment]
Committed that change. Looking at it, I'm not sure why the original `focus()` call added with this [commit](https://github.com/zenoamaro/react-quill/commit/bd2d72b930436fea2c0fb986f7312f8c6f57dc3e#diff-d24fe73020f46dde060f58786e987b61R189) was needed, as the `setSelection()` would focus or not depending if there was a cursor in the editor or not. Maybe 3 years ago quill `setSelection()` didn't behave the way it does now? 

I'm hesitant to remove the `focus()` call completely. I'm not sure if there's a possibility for `editor` to have focus without having a cursor in it, if it can then it's better to keep the `focus()` call.

**Comment by k-lusine** [comment]
Any updates?

**Comment by cjwilsontech** [comment]
@alexkrolick What's the status on getting this merged and shipped?

**Comment by fogking** [comment]
빨리 병합이 이루어졌으면 좋겠습니다.

**Comment by bjarnebonde** [comment]
Any update on getting this merged in and released?

**Comment by KoAi** [comment]
This should be merged and released, it works well

**Comment by rockwotj** [comment]
I created a fix for this according to the latest master branch [v2] here: https://github.com/zenoamaro/react-quill/pull/594.

**Comment by zenoamaro** [comment]
@rockwotj's change has been merged to master. Can you verify that it solves your issues? Cloning and `npm link` should do the trick.

**Pull request closed without merging**